File: Python/fizzbuzz.py
inp = input()
lst = inp.split()
if 1 <= int(lst[0]) < int(lst[1]) <= int(lst[2]) <= 100:
    for i in range(1,int(lst[2])+1):
        if i % int(lst[0]) == 0 and i % int(lst[1]) != 0:
            print("Fizz")
        elif i % int(lst[1]) == 0 and i % int(lst[0]) != 0:
            print("Buzz")
        elif i % int(lst[0]) == 0 and i % int(lst[1]) == 0:
            print("FizzBuzz")
        else:
            print(i)

File: C/fizzbuzz.c
#include<stdio.h>

int main(){
    int a, b, c;
    scanf("%d %d %d", &a, &b, &c);
    if (1 <= a && a < b && b <= c && c <= 100){
        for (int i = 1; i < c+1; i++){
            if (i % a == 0 && i % b != 0) printf("Fizz\n");
            else if(i % b == 0 && i % a != 0) printf("Buzz\n");
            else if(i % a == 0 && i % b == 0) printf("FizzBuzz\n");
            else printf("%d\n", i);
        }
    }
}
